### DISCUSSION TITLE
Awesome Dashboard - Had some minor tweaks for my environment

### DISCUSSION THREAD

**Discussion by tonygauderman** [bsmithio/OPNsense-Dashboard#6] [General]
I stumbled onto this dashboard having recently installed opnsense and was looking for ways to enhance opnsense.  VERY nice project!!

I found the dashboard at https://www.bsmithio.com/post/opnsense-dashboard/
That article has a couple minor issues.  First is the json file links on the site above are broken, but found them here.  2nd is that the graylog message processor order is reversed on the link above vs the github order.

Couple of adjustments I had to make to make this work in my environment.

docker-compose.yaml
remove mem_limit: 1g - my environment didn't like this, probably specific to the version of docker or docker compose I'm running
change graylog host web port to 9001 as portainer uses 9000 (obviously only affects users who are running another container on 9000)

grafana
iface variable datasource needed to change from --Grafana-- to ${dataSource} (must not be defined properly in the json??)

**Comment by bsmithio** [comment]
Hello, thank you for letting me know about these issues! I've fixed the json link, image, and updated the dashboard iface variable datasource. I've also removed the mem_limit from the docker-compose file. Glad you like the dashboard!
